test/azure-pipelines.yml

```diff
@@ -145,75 +162,75 @@ jobs:
     displayName: 'Publish test results'
 
 
-- job: 'PackageLinux'
-  timeoutInMinutes: 0
-  cancelTimeoutInMinutes: 300
-  displayName: "Build Linux Python packages"
-  pool:
-    vmImage: 'Ubuntu-16.04'
-
-  steps:
-  - script: |
-      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
-      chmod u+x dockcross-manylinux-download-cache-and-build-module-wheels.sh
-    displayName: 'Fetch build script'
-
-  - script: |
-      export ITK_PACKAGE_VERSION=$(ITKPythonGitTag)
-      ./dockcross-manylinux-download-cache-and-build-module-wheels.sh
-    displayName: 'Build Python packages'
-
-  - task: PublishPipelineArtifact@0
-    inputs:
-      artifactName: 'LinuxWheels'
-      targetPath: './dist'
-
-
-- job: 'PackageMacOS'
-  displayName: "Build macOS Python packages"
-  timeoutInMinutes: 0
-  cancelTimeoutInMinutes: 300
-  pool:
-    vmImage: 'macos-10.14'
-
-  steps:
-  - script: |
-      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/macpython-download-cache-and-build-module-wheels.sh -O
-      chmod u+x macpython-download-cache-and-build-module-wheels.sh
-    displayName: 'Fetch build script'
-
-  - script: |
-      export ITK_PACKAGE_VERSION=$(ITKPythonGitTag)
-      ./macpython-download-cache-and-build-module-wheels.sh
-    displayName: 'Build Python packages'
-
-  - task: PublishPipelineArtifact@0
-    inputs:
-      artifactName: 'MacOSWheels'
-      targetPath: './dist'
-
-
-- job: 'PackageWindows'
-  displayName: "Build Windows Python packages"
-  timeoutInMinutes: 0
-  cancelTimeoutInMinutes: 300
-  pool:
-    vmImage: 'vs2017-win2016'
-
-  steps:
-  - script: |
-      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/windows-download-cache-and-build-module-wheels.ps1 -O
-    displayName: 'Fetch build script'
-
-  - script: |
-      call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
-      set ITK_PACKAGE_VERSION=$(ITKPythonGitTag)
-      set CC=cl.exe
-      set CXX=cl.exe
-      powershell.exe -file .\windows-download-cache-and-build-module-wheels.ps1
-    displayName: 'Build Python packages'
-
-  - task: PublishPipelineArtifact@0
-    inputs:
-      artifactName: 'WindowsWheels'
-      targetPath: './dist'
+# - job: 'PackageLinux'
```

Comment:
Is packaging disabled on purpose?

Reply:
Yes.  My understanding was @thewtex was working on a different solution. 

I have re-enabled it.